heroku/cli
# migrate(W-14169186): spaces: Upgrade spaces:ps

This PR moves `spaces:ps` to the CLI and migrates to TypeScript

### Test Instructions

1. Checkout this branch and run `yarn && yarn build && cd packages/cli`
2. Run `./bin/run spaces:ps YOUR_SPACE` 
3. Verify output with the original

Note that the new `ux` package adds a newline between the header and the content of the output where it didn't exist before. This does not appear to be avoidable.
